index.js

```diff
@@ -2,39 +2,48 @@
 'use strict';
 
 var path = require('path');
+var checker = require('ember-cli-version-checker');
 var HTMLBarsInlinePrecompilePlugin = require('babel-plugin-htmlbars-inline-precompile');
 
 module.exports = {
   name: 'ember-cli-htmlbars-inline-precompile',
 
-  included: function(app) {
-    this._super.included(app);
-
-    app.options.babel = app.options.babel || {};
-    app.options.babel.plugins = app.options.babel.plugins || [];
-
-    var Compiler = require(this.templateCompilerPath());
-    var PrecompileInlineHTMLBarsPlugin = HTMLBarsInlinePrecompilePlugin(Compiler.precompile);
+  parentRegistry: null,
 
-    // add the HTMLBarsInlinePrecompilePlugin to the list of plugins used by
-    // the `ember-cli-babel` addon
-    app.options.babel.plugins.push(PrecompileInlineHTMLBarsPlugin);
+  init: function() {
+    checker.assertAbove(this, '0.1.2');
   },
 
-  // borrowed from ember-cli-htmlbars http://git.io/vJDrW
-  projectConfig: function () {
-    return this.project.config(process.env.EMBER_ENV);
+  shouldSetupRegistryInIncluded: function() {
+    return !checker.isAbove(this, '0.2.0');
```

Comment:
I think in theory, that this addon could avoid supporting < 0.2.0 of ember-cli, since this addon came out quite a while after 0.2.0 was released.

What do you think?